salt/states/service.py

```diff
@@ -255,7 +255,12 @@ def _disable(name, started, result=True, **kwargs):
         return ret
 
     # Service can be disabled
-    before_toggle_disable_status = __salt__['service.disabled'](name)
+    if salt.utils.is_windows():
+        # service.disabled in Windows returns True for services that are set to
+        # Manual start, so we need to check specifically for Disabled
+        before_toggle_disable_status = __salt__['service.info'](name)['StartType'] in ['Disabled']
```

Comment:
Can we create a function called `_salt__['service.disabled'](name) ` rather than call `__salt__['service.info'](name)['StartType']  in ['Disabled']`.  Keep in interface consistent across platforms.

Reply:
Such a function already exists, doesn't it?

Reply:
There is already a `service.disabled`, however as noted in the code comments, services that are set to start manually are considered disabled.